public/_partials/sections/Intro.jade

```diff
@@ -0,0 +1,25 @@
+#Intro.row
+  .col-xs-12
+    h1#Logo
+      img(src="assets/images/Logo.svg")
+      | PLTFRM
+    .intro.row
+      .col-xs-12.col-md-6.phones
+        img(src="assets/images/PLTRFM_TwoPhones.png")
```

Comment:
BTW it's not fully support yet (http://caniuse.com/#feat=srcset), but it's an easy win for users on modern browsers that support it.